movies/src/main/java/comp.fundacionjala/movies/Rental.java

```diff
@@ -0,0 +1,38 @@
+package comp.fundacionjala.movies;
+
+class Rental {
+
+    private IMovie movie;
+    private int daysRented;
+
+    public Rental(IMovie movie, int daysRented) {
+
+        this.movie = movie;
+        this.daysRented = daysRented;
+    }
+
+    public int getDaysRented() {
+
+        return daysRented;
+    }
+
+    public void setDaysRented(int daysRented) {
+
+        this.daysRented = daysRented;
+    }
+
+    public Movie getMovie() {
+
+        return (Movie) movie;
```

Comment:
I think this method is not necessary 
Keep in mind if you are using cast of instance, something is wrong
we need to avoid that